libkbfs/interfaces.go

```diff
@@ -2227,4 +2236,8 @@ type Chat interface {
 	ReadChannel(
 		ctx context.Context, convID chat1.ConversationID, startPage []byte) (
 		messages []string, nextPage []byte, err error)
+
+	// RegisterForMessages registers a callback that will be called
+	// for each new messages that reaches convID.
+	RegisterForMessages(convID chat1.ConversationID, cb ChatChannelNewMessageCB)
```

Comment:
Is this gonna be used by SimpleFS methods, and KBFS in the GUI will call SimpleFS methods to get a list of recent activities?

Somehow I was under the impression that FE code will use the chat messages directly. But I guess calling SimpleFS to consolidate all information makes more sense.

Reply:
> Is this gonna be used by SimpleFS methods, and KBFS in the GUI will call SimpleFS methods to get a list of recent activities?

@songgao There will be another layer or two in between SimpleFS and this chat interface, I'm working on that now.  I think we want the same info to go to both SimpleFS, the GUI, and a special `.kbfs_` file, so it should be abstracted away.  Plus my sense from GUI folks is that we want to minimize the work the GUI has to do, so I'm putting as much of the logic into the Go code that I can.